packit/source_git.py

```diff
@@ -319,9 +322,16 @@ def create_from_upstream(self):
                 f"in {self.source_git.working_dir!r}."
             )
         if not self.dist_git_specfile.uses_autosetup:
-            raise PackitException(
-                "Initializing source-git repos for packages "
-                "not using %autosetup is not supported."
+            if not self.no_require_autosetup:
+                raise PackitException(
+                    "Initializing source-git repos for packages "
+                    "not using %autosetup is not allowed by default. "
+                    "You can use --no-require-autosetup option to enforce "
+                    "running the command without %autosetup."
+                )
+            logger.warning(
+                "Source-git repos for packages not using %autosetup may be not initialized"
+                "properly or may not work with other packit commands."
```

Comment:
Please also add:

    Packit cannot guarantee that the patches will be properly applied.
    Please make sure that running %prep 
    produces a git repository with patches applied on top.
    It will be the base of your source-git repository.

Hm, that's pretty long :D should we just document it?

Reply:
Yes, this should be rather part of the docs/help-text.